vscode/webviews/chat/components/QuickStart.tsx

```diff
@@ -0,0 +1,282 @@
+import {
+    AtSignIcon,
+    ChevronDown,
+    ChevronRight,
+    type LucideProps,
+    MessageSquarePlus,
+    TextSelect,
+    X,
+    Zap,
+} from 'lucide-react'
+import type { ForwardRefExoticComponent } from 'react'
+interface ChatViewTip {
+    message: string
+    icon: ForwardRefExoticComponent<Omit<LucideProps, 'ref'>>
+    vsCodeOnly: boolean
+}
```

Comment:
Lucide exports a much more readable type for their icons 🙂 

```suggestion
    type LucideIcon,
    MessageSquarePlus,
    TextSelect,
    X,
    Zap,
} from 'lucide-react'

interface ChatViewTip {
    message: string
    icon: LucideIcon
    vsCodeOnly: boolean
}
```